src/components/search/History/SearchHistory.scss

```diff
@@ -0,0 +1,37 @@
+@import "src/styles/scaffold";
+
+.history__title {
+  align-self: center;
+  font-size: var(--text-size-3);
+  line-height: 1.5;
+}
+
+.history__header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.history__section {
```

Comment:
Do we need this class since we have the `data-store-search-history already`?

Reply:
I think you're right. I'll remove it.